bootstrap-joining-demo/local/src/main/resources/first.conf

```diff
@@ -0,0 +1,38 @@
+akka {
+
+  actor {
+    provider = "cluster"
+  }
+
+  remote {
+    artery {
+      enabled = on
+      transport = tcp
+      canonical.hostname = "akka1"
+      canonical.port = 25551
+    }
+  }
+
+  discovery {
+    method = akka-dns
+  }
+
+  management {
+    http {
+      port = 19999
+      # akka1 won't work see https://github.com/akka/akka-management/issues/187
+      hostname = "127.0.0.1"
+    }
+
+    cluster.bootstrap {
+      contact-point-discovery {
+        service-name = "akka1"
+        required-contact-point-nr = 1
```

Comment:
why 1 instead of 2?

Reply:
akka1 is the seed node so they all discover via this name